### ISSUE TITLE
Pre-work Checklist: Developer: Philip Dimaano

### ISSUE THREAD

**Issue by pdimaano** [hackforla/website#4327]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by pdimaano** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

Approximately 5-6 hours working to get to this point. My onboarding was at the end of March and there were no team meetings during the first week of April. During the developer meeting on 11APR23, Justin Dingeman also informed me that the team is no longer making "good second issues" and that I should complete another "good first issue" instead.

**Comment by HackforLABot** [comment]
## Activity Log: pdimaano
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- pdimaano opened: https://github.com/hackforla/website/issues/4327 at 03/28/2023, 08:52 PM PDT
- pdimaano assigned: https://github.com/hackforla/website/issues/4327 at 03/28/2023, 08:52 PM PDT
- pdimaano assigned: https://github.com/hackforla/website/issues/4378 at 04/03/2023, 11:17 AM PDT
- pdimaano commented on: https://github.com/hackforla/website/issues/4378#issuecomment-1495124345 at 04/03/2023, 04:34 PM PDT
- pdimaano opened: https://github.com/hackforla/website/pull/4411 at 04/04/2023, 04:22 PM PDT
- pdimaano PR was merged: https://github.com/hackforla/website/pull/4411#event-8948718508 at 04/06/2023, 11:32 AM PDT
- pdimaano issue closed by PR 4411: https://github.com/hackforla/website/issues/4378#event-8948718647 at 04/06/2023, 11:32 AM PDT
- pdimaano commented on: https://github.com/hackforla/website/issues/4327#issuecomment-1505701008 at 04/12/2023, 10:57 AM PDT
- pdimaano assigned: https://github.com/hackforla/website/issues/4448 at 04/12/2023, 11:01 AM PDT
- pdimaano commented on: https://github.com/hackforla/website/issues/4448#issuecomment-1505713005 at 04/12/2023, 11:06 AM PDT
- pdimaano opened: https://github.com/hackforla/website/pull/4517 at 04/14/2023, 02:16 PM PDT
- pdimaano PR was merged: https://github.com/hackforla/website/pull/4517#event-9048817043 at 04/19/2023, 09:46 AM PDT
- pdimaano issue closed by PR 4517: https://github.com/hackforla/website/issues/4448#event-9048817330 at 04/19/2023, 09:46 AM PDT
- pdimaano closed as completed: https://github.com/hackforla/website/issues/4327#event-9050476209 at 04/19/2023, 01:06 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/4560#issuecomment-1520525379 at 04/24/2023, 10:00 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/4560#pullrequestreview-1398492702 at 04/24/2023, 10:23 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/4560#pullrequestreview-1400727288 at 04/25/2023, 01:47 PM PDT
- pdimaano assigned: https://github.com/hackforla/website/issues/4457 at 05/07/2023, 08:35 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/issues/4457#issuecomment-1537693099 at 05/07/2023, 08:39 PM PDT
- pdimaano opened: https://github.com/hackforla/website/pull/4657 at 05/11/2023, 02:39 PM PDT
- pdimaano PR was merged: https://github.com/hackforla/website/pull/4657#event-9242242314 at 05/14/2023, 10:37 PM PDT
- pdimaano issue closed by PR 4657: https://github.com/hackforla/website/issues/4457#event-9242242414 at 05/14/2023, 10:37 PM PDT
- pdimaano assigned: https://github.com/hackforla/website/issues/4592 at 05/15/2023, 04:33 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/issues/4592#issuecomment-1548755927 at 05/15/2023, 04:37 PM PDT
- pdimaano opened: https://github.com/hackforla/website/pull/4701 at 05/19/2023, 11:48 AM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/4701#issuecomment-1604598102 at 06/23/2023, 10:27 AM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/4701#issuecomment-1612350943 at 06/28/2023, 07:31 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/4701#issuecomment-1624171782 at 07/06/2023, 11:59 AM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/4701#issuecomment-1627861268 at 07/09/2023, 04:32 PM PDT
- pdimaano PR was merged: https://github.com/hackforla/website/pull/4701#event-9846077295 at 07/17/2023, 11:20 PM PDT
- pdimaano issue closed by PR 4701: https://github.com/hackforla/website/issues/4592#event-9846077490 at 07/17/2023, 11:20 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5054#issuecomment-1648499083 at 07/24/2023, 12:37 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5054#pullrequestreview-1544522123 at 07/24/2023, 07:16 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5152#issuecomment-1668650527 at 08/07/2023, 03:20 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5152#pullrequestreview-1568095418 at 08/08/2023, 01:01 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5203#issuecomment-1679464552 at 08/15/2023, 12:16 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5203#pullrequestreview-1581682010 at 08/16/2023, 07:53 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5276#issuecomment-1686923194 at 08/21/2023, 12:34 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5276#pullrequestreview-1592513939 at 08/23/2023, 03:41 PM PDT
- pdimaano assigned: https://github.com/hackforla/website/issues/4427 at 08/29/2023, 01:31 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/issues/4427#issuecomment-1698101176 at 08/29/2023, 01:42 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5415#issuecomment-1705695074 at 09/04/2023, 01:58 PM PDT
- pdimaano opened: https://github.com/hackforla/website/pull/5444 at 09/04/2023, 07:30 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5415#pullrequestreview-1612181731 at 09/05/2023, 06:31 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5444#issuecomment-1714330932 at 09/11/2023, 10:50 AM PDT
- pdimaano PR was merged: https://github.com/hackforla/website/pull/5444#event-10366797393 at 09/13/2023, 08:47 PM PDT
- pdimaano issue closed by PR 5444: https://github.com/hackforla/website/issues/4427#event-10366797451 at 09/13/2023, 08:47 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5554#issuecomment-1726449003 at 09/19/2023, 01:51 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5554#pullrequestreview-1634420872 at 09/19/2023, 06:38 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5580#issuecomment-1734272626 at 09/25/2023, 11:37 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5580#pullrequestreview-1643071842 at 09/25/2023, 03:29 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5645#issuecomment-1743536037 at 10/02/2023, 11:29 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5645#pullrequestreview-1653598487 at 10/02/2023, 03:17 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5690#issuecomment-1755925541 at 10/10/2023, 10:34 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5690#pullrequestreview-1672706023 at 10/11/2023, 03:47 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5719#issuecomment-1765378845 at 10/16/2023, 03:37 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5719#pullrequestreview-1683865904 at 10/17/2023, 08:15 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5768#issuecomment-1776465349 at 10/23/2023, 08:31 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5768#pullrequestreview-1696137129 at 10/24/2023, 05:48 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5793#issuecomment-1785666715 at 10/30/2023, 09:59 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/5793#pullrequestreview-1705378654 at 10/30/2023, 08:26 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/5877#issuecomment-1800715805 at 11/07/2023, 04:23 PM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/5877#pullrequestreview-1721520225 at 11/08/2023, 05:01 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/5894#issuecomment-1808891600 at 11/13/2023, 11:30 AM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/5894#pullrequestreview-1731069518 at 11/14/2023, 05:06 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/5925#issuecomment-1820025326 at 11/20/2023, 04:31 PM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/5925#pullrequestreview-1743360722 at 11/21/2023, 04:17 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/5937#issuecomment-1828382357 at 11/27/2023, 10:21 AM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/5937#pullrequestreview-1751691017 at 11/27/2023, 04:50 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/5955#issuecomment-1839111266 at 12/04/2023, 09:17 AM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/6014#issuecomment-1866787266 at 12/21/2023, 10:51 AM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/6014#pullrequestreview-1793962157 at 12/21/2023, 08:38 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/6025#issuecomment-1874394929 at 01/02/2024, 10:32 AM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/6025#pullrequestreview-1805389889 at 01/04/2024, 10:06 PM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/6025#pullrequestreview-1810418647 at 01/08/2024, 09:19 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/6149#issuecomment-1908717465 at 01/24/2024, 10:43 AM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/6149#pullrequestreview-1844957728 at 01/25/2024, 05:24 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/6190#issuecomment-1915344223 at 01/29/2024, 10:42 AM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/6190#pullrequestreview-1852741683 at 01/30/2024, 05:38 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/6241#issuecomment-1928751372 at 02/05/2024, 08:09 PM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/6241#pullrequestreview-1871646696 at 02/08/2024, 08:45 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/6322#issuecomment-1953430149 at 02/19/2024, 07:29 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/6374#issuecomment-1965492442 at 02/26/2024, 03:09 PM PST
- pdimaano submitted review: https://github.com/hackforla/website/pull/6374#pullrequestreview-1907790234 at 02/28/2024, 05:53 PM PST
- pdimaano commented on: https://github.com/hackforla/website/pull/6443#issuecomment-1989001049 at 03/11/2024, 10:15 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/6443#pullrequestreview-1935426406 at 03/13/2024, 05:13 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/6518#issuecomment-2018459262 at 03/25/2024, 09:51 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/6518#pullrequestreview-1962234893 at 03/26/2024, 08:59 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/6580#issuecomment-2043657104 at 04/08/2024, 02:17 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/6580#pullrequestreview-1992846830 at 04/10/2024, 04:29 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/6711#issuecomment-2072901101 at 04/23/2024, 09:43 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/6711#pullrequestreview-2021389592 at 04/24/2024, 07:48 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/6821#issuecomment-2096734037 at 05/06/2024, 12:17 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/6821#pullrequestreview-2042075914 at 05/06/2024, 10:12 PM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/6821#pullrequestreview-2048997375 at 05/09/2024, 04:25 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/6865#issuecomment-2121441947 at 05/20/2024, 04:44 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/6897#issuecomment-2135819211 at 05/28/2024, 10:54 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/6897#pullrequestreview-2086882354 at 05/29/2024, 05:25 PM PDT
- pdimaano commented on: https://github.com/hackforla/website/pull/6980#issuecomment-2159017038 at 06/10/2024, 11:22 AM PDT
- pdimaano submitted review: https://github.com/hackforla/website/pull/6980#pullrequestreview-2116650811 at 06/13/2024, 11:39 AM PDT
